src.ts/agent/config.ts

```diff
@@ -13,29 +13,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
 
 /**
  * @typedef {object} DebugAgentConfig
  */
-
-module.exports = {
-  // FIXME(ofrobots): presently this is dependent what cwd() is at the time this
-  // file is first required. We should make the default config static.
+export interface DebugAgentConfig {
   /**
    * @property {?string}
    * @memberof DebugAgentConfig
    * @default
```

Comment:
The `@ default` doesn't make sense anymore.. how do we document the default value?